viewer/views.py

```diff
@@ -1,35 +0,0 @@
-import json
-import os.path
-
-from django.conf import settings
-from django.http import HttpResponse, HttpResponseServerError, JsonResponse
-
-from .api.models import Metric
-
-
-def ops_lbheartbeat(request):
-    return HttpResponse('ok')
-
-
-def ops_heartbeat(request):
-    try:
-        # Check the database by counting Metrics.
-        Metric.objects.count()
```

Comment:
Yep, I saw that too. I think the defaults are good enough in this case. I was only doing that count to make a database call.